NSS-Day-Cohort-49/react-nutshell-props-pushers
# Sx task

#### Changes Made

1. Added file `TaskForm` to `tasks` directory.
1. Modified file `TaskCard` to include `add, delete, and complete` functionality.
2. Modified file `TaskList` to include `add new task` functionality
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout sx-task
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
   > Example: When user does BLANK, then BLANK should happen.
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
